interfaces/cython/cantera/ctml_writer.py

```diff
@@ -798,6 +809,27 @@ def build(self, t):
         u["size"] = "9"
         u["name"] = "coeffs"
 
+class standardState(object):
+    """Base class for species standard-state properties."""
+    def _build(self, p):
+        return p.addChild("standardState")
+    def export(self, f, fmt = 'CSV'):
```

Comment:
Keyword arguments in function definitions in Python should not have spaces around the `=`. There are a few places where this happens in the code you're adding (no need to fix existing code), and if you're adding to an existing function that has spaces around the `=`, then you should match that style.

Reply:
This appears to have been resolved (except for the many many cases where it was already present :) ).